tests/test_ecograph.py

```diff
@@ -8,120 +6,94 @@
 import sklearn.preprocessing
 
 import ecoscope
-from ecoscope.analysis.ecograph import (
-    FeatureNameError,
-    IndividualNameError,
-    InterpolationError,
-)
 
 
-@pytest.mark.skip(reason="this has been failing since May 2022; will be fixed in a follow-up pull")
-def test_ecograph(movbank_relocations):
-    # apply relocation coordinate filter to movbank data
+@pytest.fixture
+def movebank_trajectory_gdf(movebank_relocations):
```

Comment:
How did we fix this test?

Reply:
The reference feather files were rebuilt. I didn't dig much into it, but at some point _something_ in rasterio changed that caused the result of `get_feature_gdf` to be different enough to break the test. It was simply a difference in the order of the geometry series, but I wasn't able to resolve it with `normalize` or `check_less_precise`, so settled for rebuilding the feathers as I'm confident this was due to a change in structure rather than a change in data.